cmd/ocm-backplane/login/login.go

```diff
@@ -50,6 +50,8 @@ var (
 		RunE:         runLogin,
 		SilenceUsage: true,
 	}
+
+	ErrLoginConnection = errors.New("unable to connect to backplane api")
```

Comment:
maybe `errLoginConnection` since the variable is not use outside the login module